lib/globals.d.ts

```diff
@@ -21,3 +21,23 @@ declare module '*.json' {
   const value: Record<string, any>;
   export = value;
 }
+
+declare module 'ssh-agent-js' {
+  class Agent {
+    constructor(socket: any);
+    add_key(key: string);
+    remove_all_keys();
+  }
+}
```

Comment:
move to types

Reply:
Sorry I don't know how. Can't put it in there as is and I don't know how I have to define it to make it work 😕

Reply:
Do it like `later`

https://github.com/renovatebot/renovate/blob/467b5ae2fe69701e7c0af2c0eaa7de446463ba40/lib/types/later.d.ts#L1

Reply:
I see - thanks. Done